lib/omniship/newgistics/track/package.rb

```diff
@@ -1,29 +1,14 @@
 module Omniship
   module Newgistics
     module Track
-      class Package
+      class Package < Omniship::Base
+        ARRIVAL_STATUSES = ["Delivered"]
+        DEPARTURE_STATUSES = ["Received", "InTransit", "Departed", "InUSPSNetwork"] + ARRIVAL_STATUSES
```

Comment:
why not? If something has arrived than you can conclude that it has also departed...